src/bridge/mod.rs

```diff
@@ -268,6 +268,34 @@ async fn start_neovim_runtime(
         .await
         .ok();
 
+    if let ConnectionMode::RemoteTcp(_) = connection_mode() {
+        // this should be default, unless disabled with `--no-clipboard`
+        // we also provide a separated setup command and disable command
+        let custom_clipboard = r#"
```

Comment:
If we are strict on code organization, having a separate code serve as the vim init for neovide is a good goal. But we should weight in performance between rpc call and script load.

Another point, as neovide ship as a standalone binary, the script is embedded. We can write the script into autoload folder on first run, and check if `g:neovide` is loaded, if `g:neovide_script_version` is lower, we update the script. Just my thoughts on this. Well be a new issue for this feature.

Placing scripts in `~/.local/share/nvim/site/autoload` or `~/.config/nvim/autoload` can make neovim loads the script on runtime, similar to how package manager works.

Reply:
Very delayed response 😅...

I like this thinking except for one issue, writing into the local machine's autoload folder would mean that those same scripts would get loaded every time afterward even if run in terminal or with another gui. This would likely have a performance burden on other systems.

I'm still thinking about this, but for now the embedded approach of manually running nvim commands on startup is sufficient.

Reply:
Another comment regarding this. We also need to be careful to load this before `init.vim`, so that users can override `g:clipboard` if they want.